modules/docs/arrow-docs/docs/docs/quickstart/libraries/README.md

```diff
@@ -77,9 +77,9 @@ advanced
 
 Advanced [typeclasses]({{ '/docs/typeclasses/intro' | relative_url }}) to be used in programs using the Tagless-final architecture.
 
-It also includes the instances available for datatypes in both arrow-core and arrow-data
+It also includes the instances available for datatypes in both arrow-core and arrow-extras
```

Comment:
Do we wanna refer to them in a generic way here? Those artifacts don't exist with that naming yet

Reply:
Yes, that's my plan